ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -1,4 +1,75 @@
 periodics:
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 0 */2 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: ocp-4.4
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
@stevekuznetsov since you don't want the `informing` label here, that means for now @ewolinetz needs to add this job to the release controller config to get testgrid-generator to include it in our dashboards, correct?  Is there any negative consequence to him doing that?

Reply:
@stevekuznetsov bump